README.md

```diff
@@ -9,11 +9,13 @@ The Cocoa UI of dupeGuru is hosted in a separate repo: https://github.com/hsoft/
 
 ## Current status
 
-Development has been slow this past year, however very close to getting all the different 4.0.4 releases posted.  Most of the work this past year (2019) has been towards packaging the application and issues related to that.
+* 2020: various bug fixes and small UI improvements have been added. Packaging for MacOS is still a problem.
+
+* 2019: Development has been slow this past year, however very close to getting all the different 4.0.4 releases posted.  Most of the work this past year (2019) has been towards packaging the application and issues related to that.
 
 Still looking for additional help especially with regards to:
 - OSX maintenance (reproducing bugs & cocoa version)
-- Linux maintenance (reproducing bugs)
+- Linux maintenance (reproducing bugs, building package with Cocoa UI)
```

Comment:
The Cocoa UI is only for OSX is this supposed to be building packages in general it would be the Qt UI on linux.

Reply:
Oops, that was meant to go on the line above.